### PR TITLE
Adds a simple output mode, helpUri and severity

### PR DESCRIPTION
**Pull request by Gby56**
This simply sets the ruleInfo and toolInfo strings to empty '' so that the output is more compact.
Also, adding automatically the severity and helpUri of each rule if found !

### CODE REVIEW

**Comment by Gby56** [comment]
@azu let me know what you think, this implements https://github.com/azu/security-alert/issues/19 :)

File: packages/sarif-to-markdown/src/sarif-to-markdown.ts [hunk 1]
```diff
@@ -121,7 +126,7 @@ export const sarifToMarkdown = (options: sarifFormatterOptions): ((sarifLog: Log
             /**
              * # Rule Info
              */
-            const ruleInfo = escapeMarkdown`
+            let ruleInfo = escapeMarkdown`
```
Comment (azu) [inline packages/sarif-to-markdown/src/sarif-to-markdown.ts:129]: I think that we can extract the task as a function like `createRuleInfo`.
If we do the refactoring, we can use `const`.
```js
const ruleInfo = simpleMode ? "" : createRuleInfo({ ... })
```

**Review by azu — COMMENTED** [review]
I've commented about code style.

LGTM about this feature.

File: packages/sarif-to-markdown/src/sarif-to-markdown.ts [hunk 2]
```diff
@@ -178,8 +195,19 @@ No Error
                     const groupContainsSuppressed =
                         group[r].filter((r: Result) => r.suppressions !== undefined).length > 0;
                     if (groupContainsSuppressed) {
+                        const ruleId = group[r][0].ruleId;
+                        const ruleMatch = run.tool.driver.rules.filter((r: any) => {
+                            return r.id == ruleId;
+                        });
+                        let severityLevel = "";
+                        let helpUri = ruleMatch[0].helpUri ?? "";
+                        if (ruleMatch[0].defaultConfiguration !== "undefined") {
```
[the remark this answers is not in the document]
Comment (Gby56) [reply packages/sarif-to-markdown/src/sarif-to-markdown.ts:204]: Thanks a lot for that, yes I wasn't sure how to check properly, probably using optional chaining + `??` that's perfect :)

[the remark this answers is not in the document]
Comment (Gby56) [reply packages/sarif-to-markdown/src/sarif-to-markdown.ts]: I implemented that nice optional chaining + nullish coalescing operator ?? :)

**Comment by azu** [comment]
Thanks! 

https://github.com/azu/security-alert/releases/tag/v1.9.0

**Comment by Gby56** [comment]
closed https://github.com/azu/security-alert/issues/19

**Pull request merged**